floris/simulation/grid.py

```diff
@@ -73,12 +77,12 @@ class Grid(ABC):
     n_wind_speeds: int = field(init=False)
     n_wind_directions: int = field(init=False)
     turbine_coordinates_array: NDArrayFloat = field(init=False)
-    x: NDArrayFloat = field(init=False, default=[])
-    y: NDArrayFloat = field(init=False, default=[])
-    z: NDArrayFloat = field(init=False, default=[])
     x_sorted: NDArrayFloat = field(init=False)
     y_sorted: NDArrayFloat = field(init=False)
     z_sorted: NDArrayFloat = field(init=False)
+    x_sorted_inertial_frame: NDArrayFloat = field(init=False)
```

Comment:
These are added and `x` etc are deleted. As I understand it, `x/y/z_sorted_intertial_frame` are the same as `x/y/z` except that they're sorted. Will we need to retain `x/y/z` for postprocessing? For example, if you need to plot the layout with ID's based on your input, do we require to unsort `x/y/z_sorted_intertial_frame`?
2.

Reply:
A follow on question is whether this should be simply `x/y/z` with those attributes being defined (in words) the same as you would define the `x/y/z_sorted_intertial_frame` attributes.